adapter/src/components/ErrorBoundary.js

```diff
@@ -48,7 +53,9 @@ export class FatalErrorBoundary extends Component {
                         </div>
                         <div
                             className="link"
-                            onClick={() => window.location.reload()}
+                            onClick={() => {
+                                this.props.onRefresh && this.props.onRefresh()
```

Comment:
Maybe `onRefresh` isn't the best name if this is to be generic - for error boundaries around individual components a "refresh" might just be reloading the component.  (this is probably a tangent, as in this case both uses result in a page refresh)

Reply:
I named it after the text that says 'Refresh to try again', but another name is also fine :)

Now that I'm thinking about it, that text might be missing some affordance that attracts user interaction - it sounds a bit like an instruction that might get a user to go and user their browser refresh button instead of clicking on it, and it doesn't look like a button or link that you see in apps. Anecdotally, I didn't know this link did anything for a long time 😅 It would probably benefit from getting @cooper-joe to take a look!

In the mean time, maybe 'Click here to try again' might be a small improvement, and the prop name could change too: 'onRetry' maybe?

Reply:
Thanks for the ping! I think it would be best to change this over to a `Button` (type _basic_), with text `Try again`. This invites user interaction and accurately describes the intention of the action, rather than the action itself.

If we don't have access to the `Button`, or we want to stick with a link for now, then `Click here to try again` is a good quickfix 👍.

Reply:
Thanks Joe! 🙏

Reply:
We tried to avoid using `ui` components in the fatal error boundary as anything that breaks within it will not be caught properly, but we could use a standard `<button>` with some styling

Reply:
Anything with an `onClick` should be a button in order to be keyboard accessible so I'm also in favour of using a `<button>` with some (additional to the existing) css like

```css
.button {
  border: none;
  background: transparent;
}
```

Also it'd be nice if the 'show technical' details button was changed from a `div` to a `button` too.

Then again, the rest of the error boundary component could do with some TLC, such as increasing the default height & width of the stack trace drawer, using `colors-red600` instead of `red`, and adding a "copy" button in order to make reporting such errors easier for our users - so maybe these changes should be done in a separate PR?

Reply:
Yeah those would be great, but let's make them in a separate PR